valohai_yaml/objs/step.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         environment_variables: Iterable[EnvironmentVariable] = (),
         environment: Optional[str] = None,
         description: Optional[str] = None,
+        custom_store: Optional[str] = None,
```

Comment:
Considering the fact that execution should not run with invalid store, renaming field to upload-store makes everything much easier

Reply:
Maybe it is even better to use only store name instead of allowing both name and uuid? Store UUID is not easily found on frontend and it is always more chances to may a mistake with it.